README.md

```diff
@@ -119,19 +112,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Get issue data
+      - name: Get User Input
+        id: get_input
+        uses: jasonmacgowan/parse-issue@master
+        with:
+          extract_email: '<p>Email of Requester:\s*(.*)</p>'
+        env:
+          ADMIN_TOKEN: ${{ secrets.ADMTIN_TOKEN }}
```

Comment:
Let's remove this or fix secrets reference to ADMIN_TOKEN

Reply:
apologies, don't know why I marked it as resolved if I didn't do anything about it.  
I will get it removed today.